modules/apps/frontend-taglib/frontend-taglib-clay/src/main/resources/META-INF/resources/management_toolbar/ResultsBar.js

```diff
@@ -87,7 +109,7 @@ const ResultsBar = ({
 							searchValue
 						)}
 						className="component-link tbar-link"
-						href={clearResultsURL}
+						onClick={(event) => handleClick(event)}
```

Comment:
We can simplify this to`onClick={handleClick}`. 

As `handleClick` is small, we can even move it inline.